dist/slant-frame.js

```diff
@@ -532,10 +554,12 @@ Frame.prototype.onclick = function (e) {
     e.stopPropagation();
   }
 
-  if (this.state.playing) {
-    this.pause();
-  } else {
-    this.play();
+  if (false == this.state.image) {
```

Comment:
instead of wrapping all of these functions in if-statements, it could be cleaner to do something like this: 

if (true === this.state.image) return;